packages/dashboard/src/components/tasks/task-schedule.tsx

```diff
@@ -179,7 +179,7 @@ export const TaskSchedule = () => {
 
       let exceptDate: string | undefined = undefined;
       if (eventScope === EventScopes.CURRENT) {
-        exceptDate = exceptDateRef.current.toISOString();
+        exceptDate = schedule.at.toISOString();
```

Comment:
I believe this change is not needed, as it will set `exceptDate` to today on the schedule's date, instead of the one selected for that event. You'll see it bugging out if you try to modify or delete an event next week, instead of today or tomorrow

Reply:
You're right!
Thanks!